src/game/client/components/menus_demo.cpp

```diff
@@ -401,6 +402,18 @@ void CMenus::DemolistOnUpdate(bool Reset)
 	m_DemolistSelectedIsDir = m_DemolistSelectedIndex < 0 ? false : m_lDemos[m_DemolistSelectedIndex].m_IsDir;
 }
 
+bool CMenus::FetchHeader(CDemoItem &Item)
```

Comment:
You should pass by pointer https://github.com/teeworlds/documentation/blob/master/nomenclature.md#passing-variables